src/client/mvt/feature-decoder.js

```diff
@@ -28,12 +28,24 @@ export function decodeLines(geom, mvt_extent) {
 export function decodePolygons(geometries, mvtExtent) {
     let currentPolygon = null;
     let decoded = [];
+    let invertedOrientation;
+    geometries.forEach(geom => {
+        let area = signedPolygonArea(geom);
+        if (area == 0) {
+            return;
+        }
+        if (invertedOrientation === undefined) {
+            // According to the MVT spec this condition cannot happen for
+            // MVT spec compliant tiles, but many buggy implementations
+            // don't comply with this rule when generating tiles
+            // Also, other implementations accept this out-of-the-spec condition
+            invertedOrientation = area > 0;
+        }
+        const isExternalPolygon = invertedOrientation ? area > 0 : area < 0;
 
```

Comment:
I am a bit confused by this code, I think I am missing something. I think this is the same than: 

```js
let area = signedPolygonArea(geom);

if (area === 0) {
  return;
}

const isExternalPolygon = invertedOrientation !== undefined
  ? area > 0 
  : area < 0;
```